src/test/scala/treadle/GetIntBreakdownSpec.scala

```diff
@@ -53,7 +53,7 @@ class GetIntBreakdownSpec extends FreeSpec with Matchers {
     }
     true
   }
-  "GetIntBreakdownSpec should pass a basic test" in {
+  "GetIntBreakdownSpec should pass a basic test" ignore {
```

Comment:
Is this test intentionally ignored (as opposed to a debugging change that was unintentionally committed)?